projects/HW102/src/main.c

```diff
@@ -0,0 +1,37 @@
+#include <stdint.h>
+#include "interrupt.h"
+#include "log.h"
+#include "soft_timer.h"
+#include "wait.h"
+
+#define SECOND 1000
+#define HALF_SECOND 500
```

Comment:
It's great that you factored these out into defines! Usually though we use names that don't just describe the constant itself but its purpose, so mentioning that it's a timeout or something would be better than just `SECOND`. The convention is also to add a suffix with the unit, so `_MS`.